internal/core/src/segcore/plan_c.cpp

```diff
@@ -59,16 +59,18 @@ ParsePlaceholderGroup(CSearchPlan c_plan,
                       const void* placeholder_group_blob,
                       const int64_t blob_size,
                       CPlaceholderGroup* res_placeholder_group) {
-    auto plan = (milvus::query::Plan*)c_plan;
+    auto plan = static_cast<milvus::query::Plan*>(c_plan);
 
     try {
         auto res = milvus::query::ParsePlaceholderGroup(
-            plan, (const uint8_t*)(placeholder_group_blob), blob_size);
+            plan,
+            static_cast<const uint8_t*>(placeholder_group_blob),
+            blob_size);
 
         auto status = CStatus();
         status.error_code = milvus::Success;
         status.error_msg = "";
-        auto group = (CPlaceholderGroup)res.release();
+        auto group = static_cast<CPlaceholderGroup>(res.release());
```

Comment:
same, is it really `CPlaceholderGroup` and not `CPlaceholderGroup*`?

Reply:
Same as above.
CPlaceholderGroup is defined as void *.
`typedef void* CPlaceholderGroup;`